packages/web/src/components/gcds-file-uploader/readme.md

```diff
@@ -25,14 +25,14 @@
 
 ## Events
 
-| Event                    | Description                                    | Type                  |
-| ------------------------ | ---------------------------------------------- | --------------------- |
-| `gcdsBlur`               | Emitted when the uploader loses focus.         | `CustomEvent<void>`   |
-| `gcdsError`              | Emitted when the input has a validation error. | `CustomEvent<object>` |
-| `gcdsFileUploaderChange` | Update value based on user selection.          | `CustomEvent<any>`    |
-| `gcdsFocus`              | Emitted when the uploader has focus.           | `CustomEvent<void>`   |
-| `gcdsRemoveFile`         | Remove file and update value.                  | `CustomEvent<any>`    |
-| `gcdsValid`              | Emitted when the input has a validation error. | `CustomEvent<object>` |
+| Event            | Description                                    | Type                  |
+| ---------------- | ---------------------------------------------- | --------------------- |
+| `gcdsBlur`       | Emitted when the uploader loses focus.         | `CustomEvent<void>`   |
+| `gcdsChange`     | Update value based on user selection.          | `CustomEvent<any>`    |
```

Comment:
```suggestion
| `gcdsChange`     | Emitted when the user has made a file selection.          | `CustomEvent<any>`    |
```

Would this be a better description? I know this readme is autogenerated but we could change it in the comment